Templates/GemRepo/Template/gem.json

```diff
@@ -1,19 +1,23 @@
 {
-    "gem_name": "${Name}Gem",
-    "display_name": "${Name}Gem",
-    "license": "What license ${Name}Gem uses goes here: i.e. Apache-2.0 Or MIT",
-    "license_url": "",
-    "origin": "The primary repo for ${Name}Gem goes here: i.e. http://www.mydomain.com",
-    "summary": "A short description of ${Name}Gem which is zipped up in an archive named gem.zip in the root of the Gem Repo. Though not required, it is recommended that the sha256 of the gem.zip file should be placed in the sha256 field of this gem.json so the download can be verified.",
+    "gem_name": "${Name}",
+    "display_name": "${Name}",
+    "license": "What license ${Name} uses goes here: i.e. Apache-2.0 or MIT",
+    "license_url": "Link to the license web site goes here: i.e. https://opensource.org/licenses/Apache-2.0 Or https://opensource.org/licenses/MIT",
+    "origin": "The name of the originator goes here. i.e. XYZ Inc.",
+    "origin_url": "The primary repo for ${Name} goes here: i.e. http://www.mydomain.com",
+    "type": "Code",
+    "summary": "A short description of ${Name} which is zipped up in an archive named gem.zip in the root of the Gem Repo. Though not required, it is recommended that the sha256 of the gem.zip file be placed in the sha256 field of this gem.json so the download can be verified.",
     "origin_uri": "${RepoURI}/gem.zip",
     "sha256": "",
-    "type": "Code",
     "canonical_tags": [
         "Gem"
     ],
     "user_tags": [
-        "${Name}Gem"
+        "${Name}"
     ],
     "icon_path": "preview.png",
-    "requirements": ""
+    "requirements": "",
+    "documentation_url": "",
+    "dependencies": [
+    ]
```

Comment:
${Name} refers to the name of the gem repo, not necessarily the gem, which is why these were ${Name}Gem, but it's probably OK for them to be the same.

Reply:
Yeah its fine